src/utils/sqs_consumer.ts

```diff
@@ -1,3 +1,4 @@
+// tslint:disable: max-classes-per-file
```

Comment:
Team,

Start keeping a list (in your head or somewhere) of arbitrary rules like this that aren't serving us well. I've been chatting with @xianny about a migration to eslint and that will be a good time to get rid of things like this we just end up turning off.

-d

Reply:
totally - can also do a search across the codebase for `tsling:disable` to see what types are prevalent. But I think the max classes per file one is a rather annoying restriction